src/types.ts

```diff
@@ -172,3 +172,77 @@ export type FlowAuthentication = {
   /** The refresh JWT from an active descope session */
   refreshJwt: string
 }
+
+/** Provide options when embedding a FlowView into your app */
+export type FlowOptions = {
+  android: AndroidFlowOptions
+  iOS: iOSFlowOptions
+}
+
+/** Android specific flow options */
+export type AndroidFlowOptions = {
+  /** The URL where the flow is hosted */
+  url: string
```

Comment:
I wonder where do we pass other properties of flows like style, logger, etc?

on other words, where doe the type of the web component input comes from?

Reply:
Not entirely sure what you mean here. But take into account that not everything is supported ATM.